src/core/thread/announce_sender.hpp

```diff
@@ -95,23 +111,76 @@ class AnnounceSenderBase : public InstanceLocator, private NonCopyable
     bool IsRunning(void) const { return mTimer.IsRunning(); }
 
     /**
-     * This method gets the period for the latest scheduled MLE Announce transmission (the one in progress or the last
-     * finished one).
+     * This method gets the period interval.
      *
-     * @returns The period interval (in milliseconds) between two successive MLE Announcement transmissions.
+     * @returns The period interval (in milliseconds).
      *
      */
     uint32_t GetPeriod(void) const { return mPeriod; }
 
     /**
-     * This method gets the channel mask for the latest scheduled MLE Announce transmission (the one in progress or the
-     * last finished one).
+     * This method sets the period interval.
+     *
+     * The period along with jitter value from (`Get/SetJitter()`) determines the interval between two successive MLE
+     * Announcement transmissions (possibly) on different channels from the specified channel mask.
+
+     * @param[in] aPeriod   The period interval (in milliseconds).
+     *
+     */
+    void SetPeriod(uint32_t aPeriod) { mPeriod = aPeriod; }
+
+    /**
+     * This method gets the current jitter interval.
      *
-     * @returns A constant reference to channel mask
+     * @returns The jitter interval (in milliseconds).
+     *
+     */
+    uint16_t GetJitter(void) const { return mJitter; }
```

Comment:
Not used. Remove this?

Reply:
I think it would be good to keep getter when we have a setter for the property. It's small and inline and if not used will be linked out.